drivers/console/uart_mux.c

```diff
@@ -812,7 +812,7 @@ int uart_mux_send(const struct device *uart, const uint8_t *buf, size_t size)
 
 	k_mutex_unlock(&dev_data->real_uart->lock);
 
-	return 0;
+	return size;
```

Comment:
but this will always return zero because of `while (--size)` above

Reply:
err.. sorry... didn't have enough caffeine in my brain. fixed.